tasks/terraform-cli/src/task.ts

```diff
@@ -109,4 +110,13 @@ export class Task {
     fmt(): commands.ICommand {
         return new commands.FormatCommandHandler(this.runner);
     }
+
+    workspace(): commands.ICommand {
+        switch(this.ctx.workspaceSubCommand){
+            case "select":
+                return new commands.WorkspaceSelectCommandHandler(this.runner);
+            default:
+                throw new Error(`Workspace sub-command "${this.ctx.workspaceSubCommand}" not supported`);

```

Comment:
Also nit-picking :) `is not supported.`?
```suggestion
                throw new Error(`Workspace sub-command "${this.ctx.workspaceSubCommand}" is not supported`);

```

Reply:
another good catch. fixed!